### PR TITLE
Bug 1935087: Align Arbiter text with the new behaviour introduced by BZ #1928319

### PR DESCRIPTION
**Pull request by SanjalKatiyar**
After BZ: 1928319, we are now utilising attached LSO devices in the multiple of 4, so, the text that we show on step-3 for the arbiter configuration needs to be updated as well.

**Earlier (for arbiter cases):**
Screenshot_20210517_193909

**After (for arbiter cases):**
Screenshot from 2021-06-01 13-12-28

**Earlier (for non-arbiter cases):**
Screenshot from 2021-06-01 12-46-52

**After (for non-arbiter cases):**
Screenshot from 2021-06-01 13-06-28

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json [hunk 1]
```diff
@@ -532,7 +532,7 @@
...
   "Selected nodes are based on the selected storage class. The selected nodes will preferably be in 3 different zones with a recommended requirement of 14 CPUs and 34 GiB per node.": "Selected nodes are based on the selected storage class. The selected nodes will preferably be in 3 different zones with a recommended requirement of 14 CPUs and 34 GiB per node.",
-  "Selected nodes are based on the selected storage class. 4 nodes in 2 different zones will be selected with a recommended requirement of 14 CPU and 34 GiB RAM per node.": "Selected nodes are based on the selected storage class. 4 nodes in 2 different zones will be selected with a recommended requirement of 14 CPU and 34 GiB RAM per node.",
+  "Selected nodes are based on the selected storage class. Minimum 4 nodes in 2 different zones will be selected with a recommended requirement of 14 CPU and 34 GiB RAM per node.": "Selected nodes are based on the selected storage class. Minimum 4 nodes in 2 different zones will be selected with a recommended requirement of 14 CPU and 34 GiB RAM per node.",
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json:535]:
```suggestion
  "Selected nodes are based on the selected storage class. Minimum 4 nodes in 2 different zones will be selected with a recommended requirement of 14 CPU and 34 GiB RAM per node.": "Selected nodes are based on the selected storage class and the arbiter configuration requirements."
  thoughts ??

Comment (SanjalKatiyar) [reply frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json:535]: Looks good, let me confirm it with UX.

**Review by afreen23 — APPROVED** [review]
/lgtm

File: frontend/packages/ceph-storage-plugin/src/constants/tooltips.ts [hunk 2]
```diff
@@ -16,13 +16,15 @@ export const arbiterText = (t: TFunction) =>
...
-    'ceph-storage-plugin~Selected nodes are based on the selected storage class. The selected nodes will preferably be in 3 different zones with a recommended requirement of 14 CPUs and 34 GiB per node.',
+    'ceph-storage-plugin~Selected nodes are based on the storage class {{storageClassName}} and with a recommended requirement of 14 CPU and 34 GiB RAM per node.',
+    { storageClassName: scName },
   );
-export const attachDevicesWithArbiter = (t: TFunction) =>
+export const attachDevicesWithArbiter = (t: TFunction, scName: string) =>
   t(
-    'ceph-storage-plugin~Selected nodes are based on the selected storage class. 4 nodes in 2 different zones will be selected with a recommended requirement of 14 CPU and 34 GiB RAM per node.',
+    'ceph-storage-plugin~Selected nodes are based on the storage class {{storageClassName}} and fulfill the stretch cluster requirements with a recommended requirement of 14 CPU and 34 GiB RAM per node.',
+    { storageClassName: scName },
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/constants/tooltips.ts:27]: If you are getting missing i18n key errors in console logs then use `<Trans/>`
```suggestion
    'ceph-storage-plugin~Selected nodes are based on the storage class <em>{{storageClassName}}</em> and fulfill the stretch cluster requirements with a recommended requirement of 14 CPU and 34 GiB RAM per node.',
    { storageClassName: scName },
```

File: frontend/packages/ceph-storage-plugin/src/constants/tooltips.ts [hunk 3]
```diff
@@ -16,13 +16,15 @@ export const arbiterText = (t: TFunction) =>
   t(
     'ceph-storage-plugin~If you wish to use the Arbiter stretch cluster, a minimum of 4 nodes (2 different zones, 2 nodes per zone) and 1 additional zone with 1 node is required. All nodes must be pre-labeled with zones in order to be validated on cluster creation.',
   );
-export const attachDevices = (t: TFunction) =>
+export const attachDevices = (t: TFunction, scName: string) =>
   t(
-    'ceph-storage-plugin~Selected nodes are based on the selected storage class. The selected nodes will preferably be in 3 different zones with a recommended requirement of 14 CPUs and 34 GiB per node.',
+    'ceph-storage-plugin~Selected nodes are based on the storage class {{storageClassName}} and with a recommended requirement of 14 CPU and 34 GiB RAM per node.',
+    { storageClassName: scName },
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/constants/tooltips.ts:22]: Same here, make storage class name emphasized

**Review by afreen23 — CHANGES_REQUESTED** [review]

**Review by afreen23 — APPROVED** [review]
/lgtm

**Pull request merged**